code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -50,7 +50,7 @@
 		C.adjustOxyLoss(-2 * delta_time)
 		C.adjustCloneLoss(-2 * delta_time)
 		return
-	C.blood_volume -= 0.125 * delta_time
+	C.blood_volume -= BLOOD_REGEN_FACTOR  * 1.375 * delta_time //snowflake code so vamps do not regen blood.
```

Comment:
but now vampires who've consumed ghoul powder, are cold, are in a regenerative coma, etc. will lose blood faster

more importantly, vampires who're above BLOOD_VOLUME_NORMAL (which you'll almost always be after you drain a monkey or two) will lose blood at a faster rate than normal

also, have you checked to see if this will cause your blood level thing in the status menu to fluctuate up and down?

Reply:
> also, have you checked to see if this will cause your blood level thing in the status menu to fluctuate up and down?

will test that later, they _should_ be on the same tick.

I really don’t wanna make a trait that only one janky race uses.